metricbeat/module/kubernetes/state_container/_meta/fields.yml

```diff
@@ -26,14 +26,14 @@
     - name: cpu
       type: group
       fields:
-        - name: limit.nanocores
+        - name: limit.cores
```

Comment:
As `state_container` is already GA we should not do breaking changes. Could we perhaps keep both fields but mark one as deprecated? Same below.

Reply:
I've pushed a change keeping them (deprecated)